aspnetcore.yaml

```diff
@@ -2,9 +2,9 @@ steps:
 - name: 'gcr.io/gcp-runtimes/aspnetcorebuild:latest'
```

Comment:
There seems to be a lot of common code between all of these yaml files, anything we can do to reduce this?

Reply:
There's really no good way to share code between them, but we could have a generator perhaps.